cd-scripts/cdRelease.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
could we write this in python instead?  (or any sane language, although python is the obvious choice in this context...but my main motivation here is bash-avoidance rather than python-suprematism)

Reply:
Good point, I'll rewrite this in Python. (Although I don't hate bash 😊)

Reply:
Done 🥳